packages/trpc/server/routers/viewer/teams/create.handler.ts

```diff
@@ -17,11 +15,22 @@ type CreateOptions = {
 };
 
 export const createHandler = async ({ ctx, input }: CreateOptions) => {
+  const { user } = ctx;
   const { slug, name, logo } = input;
+  const isOrgChildTeam = !!user.organizationId;
+
+  // For orgs we want to create teams under the org
+  if (user.organizationId && !user.organization.isOrgAdmin) {
+    throw new TRPCError({ code: "FORBIDDEN", message: "org_admins_can_create_new_teams" });
+  }
+
   const currentOrgId = ctx.user.organization?.id;
+
   const slugCollisions = await prisma.team.findFirst({
     where: {
       slug: slug,
+      // If this is under an org, check that the team doesn't already exist
+      ...(isOrgChildTeam && { parentId: user.organizationId }),
       parentId: currentOrgId
```

Comment:
Isn't it doing the same thing that is at line 34. `currentOrgId` is equal to user.organization?.id

Reply:
There was an instance where @sean-brydon and I were working on the same thing, so had to clean up the file. I already talked to Sean about it.